DuckDuckGo/DoNotSellSettingsViewController.swift

```diff
@@ -0,0 +1,120 @@
+//
+//  DoNotSellSettingsViewController.swift
+//  DuckDuckGo
+//
+//  Copyright © 2020 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import UIKit
+import Core
+
+class DoNotSellSettingsViewController: UITableViewController {
+
+    @IBOutlet var labels: [UILabel]!
+    
+    @IBOutlet weak var doNotSellToggle: UISwitch!
+    @IBOutlet weak var headerView: UIView!
+    @IBOutlet weak var footerView: UIView!
+    @IBOutlet weak var infoTextView: UITextView!
+    @IBOutlet weak var disclaimerTextView: UITextView!
+    
+    private lazy var appSettings = AppDependencyProvider.shared.appSettings
+    
+    let learnMoreStr = "Learn More"
+    
+    override func viewDidLoad() {
```

Comment:
In general all the blurb text's style doesn't match what is in Figma - if I'm looking at the right [place](https://www.figma.com/file/g1Ucu8mVgtHfVrNzrKyKUK/Global-Privacy-Control--Apps-and-Extensions?node-id=214%3A1294).  Font size, color and line spacing all appear to be off.  

In Figma you can get some tips about line spacing by clicking the text and choosing  iOS from the code drop down.  

You may need to work backwards from Figma to determine which built in color to use (shouldn't be adding any new ones), though you may find it makes sense to add a couple of new theme attributes.

Once you've made the changes, a matter of process (which tbf you didn't know about), can you share some screenshots in both light and dark mode with Mike in a sub task and get him to give them a thumbs up, please?